### PR TITLE
OIDC: 'aud' claims can be arrays

### LINKED ISSUE #331: OpenIDConnect 'aud' claim is generally an array
Hi. This line in the `validateClaims` function assumes that the 'aud' claim is a string:

https://github.com/yiisoft/yii2-authclient/blob/e9e4ce479b7c9ea4f6a90ce015e362515f4723a6/src/OpenIdConnect.php#L447

However the [OIDC spec](https://openid.net/specs/openid-connect-core-1_0.html#IDToken) says that this is tolerated, but that it is generally an array:

> aud:      REQUIRED. Audience(s) that this ID Token is intended for. It MUST contain the OAuth 2.0 client_id of the Relying Party as an audience value. It MAY also contain identifiers for other audiences. **In the general case, the aud value is an array of case sensitive strings. In the common special case when there is one audience, the aud value MAY be a single case sensitive string.** 

I suggest supporting arrays.

### PR DESCRIPTION
**Pull request by azmeuk**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | Fixes #331

### CODE REVIEW

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Please add a line for CHANGELOG. Thanks.

**Comment by azmeuk** [comment]
Done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Yii Framework 2 authclient extension Change Log
 ------------------------
 
 - Bug #330: OpenID Connect client now defaults to `'client_secret_basic'` in case `token_endpoint_auth_methods_supported` isn't specified (rhertogh)
-
+- Bug #332: OpenID Connect `aud` claim can either be a string or a list of strings.
```
Comment (samdark) [inline CHANGELOG.md:8]:
```suggestion
- Bug #332: OpenID Connect `aud` claim can either be a string or a list of strings (azmeuk)
```

Comment (azmeuk) [reply CHANGELOG.md]: Done.

**Comment by samdark** [comment]
Thanks.

**Pull request merged**